chord_metadata_service/discovery/stats.py

```diff
@@ -60,18 +60,15 @@ async def bento_public_format_count_and_stats_list(
 
     async for q in annotated_queryset:
         label = q["label"]
-        value = thresholded_count(int(q["value"]), discovery, low_counts_censored)
+        raw_value = int(q["value"])
+        thresholded_value = thresholded_count(int(q["value"]), discovery, low_counts_censored)
```

Comment:
reuse `raw_value` as the first param to avoid doing the index-access + int-cast twice